tools/src/process-allure-reports.py

```diff
@@ -0,0 +1,101 @@
+import os
+import subprocess
+import argparse
+from allure_combine import combine_allure
+
+COMBINE_DIR = 'combine'
+COMPLETE_FILE_NAME = 'index.html'
+PUT_TIMEOUT = 600
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Process allure reports')
+    parser.add_argument('--neofs_node', required=True, type=str, help='NeoFS node')
+    parser.add_argument('--wallet', required=True, type=str, help='Path to the wallet')
+    parser.add_argument('--cid', required=True, type=str, help='Container ID')
+    parser.add_argument('--run_id', required=True, type=str, help='GitHub run ID')
+    parser.add_argument('--allure_report', type=str, help='Path to generated allure report directory',
+                        default='allure_report')
+    return parser.parse_args()
+
+
+def put_combine_result_as_static_page(directory: str, neofs_node: str, wallet: str, cid: str, run_id: str,
+                                      password: str) -> None:
+    base_cmd = (
+        f'NEOFS_CLI_PASSWORD={password} neofs-cli --rpc-endpoint st1.{neofs_node}:8080 '
```

Comment:
so according to this `neofs_node` isn't really a node endpoint but particular NeoFS network domain. Don't u think it may cause some misunderstanding during args' setting?